neqo-transport/src/ecn.rs

```diff
@@ -125,6 +127,36 @@ impl AddAssign<IpTosEcn> for EcnCount {
     }
 }
 
+#[derive(PartialEq, Eq, Debug, Clone, Copy, Default)]
+pub struct EcnValidationCount(EnumMap<EcnValidationOutcome, u64>);
```

Comment:
Related: Following convention within this `mod` by prefixing every `pub` type with the module name, i.e. `EcnXXX`.

I prefer not doing this, but instead follow:

https://rust-lang.github.io/rust-clippy/master/index.html#module_name_repetitions

Which we currently disable:

https://github.com/mozilla/neqo/blob/a7581770ff138c16727e92ecb7b47d9ceb95425e/neqo-transport/src/lib.rs#L7

In case there is consensus, happy to do the rename in a follow-up pull request.

Reply:
I'm fine with removing the `allow` and do the required renames. (I'm in general for reducing clippy `allow`s.)